plugins/modules/cloud/scaleway/scaleway_database_backup.py

```diff
@@ -0,0 +1,346 @@
+        - UUID used to identify the database backup.
+        - Required for C(absent), C(exported) and C(restored) states
+    type: str
+    required: false
+
+  name:
+    description:
+        - Name used to identify the database backup.
+        - Required for C(present) state
+    type: str
+    required: false
+
+  database_name:
+    description:
+        - Name used to identify the database.
+        - Required for C(present) and C(restored) states
+    type: str
+    required: false
+
+  instance_id:
+    description:
+        - UUID of the instance associated to the database backup
+        - Required for C(present) and C(restored) states
+    type: str
+    required: false
+
+  expires_at:
+    description:
+        - Expiration datetime of the database backup (ISO 8601 format)
+    type: str
+    required: false
+
+  wait:
+    description:
+        - Wait for the instance to reach its desired state before returning.
+    type: bool
+    default: 'no'
+
+  wait_timeout:
+    description:
+        - Time to wait for the backup to reach the expected state
+    type: int
+    required: false
+    default: 300
+
+  wait_sleep_time:
+    description:
+        - Time to wait before every attempt to check the state of the backup
+    type: int
+    required: false
+    default: 3
+'''
+
+EXAMPLES = '''
+  - name: Create a backup
+    scaleway_database_backup:
+        name: 'my_backup'
+        state: present
+        region: 'fr-par'
+        database_name: 'my-database'
+        instance_id: '50968a80-2909-4e5c-b1af-a2e19860dddb'
+
+  - name: Export a backup
+    scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: exported
+        region: 'fr-par'
+
+  - name: Restore backup
+    scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: restored
+        region: 'fr-par'
+        database_name: 'my-new-database'
+        instance_id: '50968a80-2909-4e5c-b1af-a2e19860dddb'
+'''
+
+RETURN = '''
+data:
+    description: This is only present when C(state=present)
+    returned: when C(state=present)
+    type: dict
+    sample: {
+        "data": {
+            "created_at": "2020-08-06T12:42:05.631049Z",
+            "database_name": "my-database",
+            "download_url": null,
+            "download_url_expires_at": null,
+            "expires_at": null,
+            "id": "a15297bd-0c4a-4b4f-8fbb-b36a35b7eb07",
+            "instance_id": "617be32e-6497-4ed7-b4c7-0ee5a81edf49",
+            "instance_name": "my-instance",
+            "name": "backup_name",
+            "region": "fr-par",
+            "size": 600000,
+            "status": "ready",
+            "updated_at": "2020-08-06T12:42:10.581649Z"
+        }
+    }
+'''
+
+import datetime
+import time
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.scaleway import (
+    Scaleway, scaleway_argument_spec,
+    SCALEWAY_REGIONS)
```

Comment:
```suggestion
    Scaleway,
    scaleway_argument_spec,
    SCALEWAY_REGIONS,
)
```
just a good practice